### ISSUE TITLE
Huawei Devices gets an error

### ISSUE THREAD

**Issue by Lawati97** [jaumard/sms_autofill#155]
For some reason when I get platform exception when am using this package with Huawei non gms devices

**Comment by Apoorvgarg-creator** [comment]
Facing the same issue, It is occurring on Android version 13, How to solve this issue ?

**Comment by danuprakoso26** [comment]
> Facing the same issue, It is occurring on Android version 13, How to solve this issue ?

did you find any solutions?

**Comment by Apoorvgarg-creator** [comment]
> > Facing the same issue, It is occurring on Android version 13, How to solve this issue ?
> 
> did you find any solutions?

Yes, Android 13 now uses Sms user consent Api and this package depends upon Sms retriever Api. So by implementing the former one, you can solve the issue

**Comment by kailashndx** [comment]
@Apoorvgarg-creator can you explain the solution little bit

**Comment by kailashndx** [comment]
@Apoorvgarg-creator can you please explain the solution little bit
